docs/source/openapi.json

```diff
@@ -774,7 +773,7 @@
             "type": "integer"
           }
         },
-        "required": ["src", "height", "width"]
+        "required": ["src"]
```

Comment:
Hmmm, note that it comes from

https://github.com/huggingface/dataset-viewer/blob/539b43775760f68f48d6df252ebef6e14c94edb6/libs/libcommon/src/libcommon/viewer_utils/asset.py#L20

It means that data coming from the API will always contain height and width. If you want to set them as optional on the Hub, maybe do so outside of the openapi.d.ts file.

Reply:
ahhh, makes sense. will revert this.

Reply:
wouldn't this be the same for audio type? 

We would expect that the API would only return this?

```python
              "enum": ["audio/wav", "audio/mpeg", "audio/ogg"]
```

and we need to make new type on moonlanding?

Reply:
no, I checked in the code, it can be any extension after all